.travis.yml

```diff
@@ -1,13 +1,15 @@
-sudo: required
-dist: trusty
+os:
+  - linux
+  - osx
+language: ruby
 before_install:
-  - gem install bundler
+  - gem install bundler -v '< 2'
 rvm:
-  - 2.5.0
-  - 2.4.3
-  - 2.3.6
-  - 2.2.9
-  - 2.1.10
+  - 2.5
+  - 2.4
+  - 2.3
+  - 2.2
+  - 2.1
```

Comment:
Unfortunately, we specify the full version on "purpose" to avoid issues with
either Travis or RVM not using the last release for each branch :-(

2.5: built with 2.5.1, should be 2.5.3
2.4: built with 2.4.4, should be 2.4.5

It means we do have to specify and regularly update the "teeny" version number.
Unless we find a working solution of course.

Please see #213 which contain interesting info and feedback about this issue,
but also a little about the Travis infrastructure selection.